charts/atlantis/templates/statefulset.yaml

```diff
@@ -399,6 +417,11 @@ spec:
             readOnly: true
             mountPath: /etc/secret-gitconfig
           {{- end }}
+          {{- if or .Values.netrc .Values.netrcSecretName }}
+          - name: netrc-volume
+            readOnly: true
+            mountPath: /etc/secret-netrc
```

Comment:
Why not just mount it `/home/atlantis/.netrc`

Reply:
I suppose this is following the same method as the gitconfig so it's probably fine.

Reply:
@nitrocode - I just tested both the gitconfig and the new netrc setup with a direct mount instead of using the lifecycle block. Seems to be working just fine. Was there a reason the gitconfig wasn't directly mounted in the first place? If not I'll push a change to directly mount instead.

Reply:
Ran a few tests this week and didn't hit any issues so I just pushed up the direct mount

Reply:
You can set this env var instead

https://www.runatlantis.io/docs/server-configuration.html#write-git-creds

Reply:
That doesn't fit my use case for a few reasons:
1. It only writes out the creds for the baseURL that's set. This doesn't solve my problem of pulling modules from Artifactory.
2. In some use cases we need to use a different set of credentials so we can authenticate with vendor provided module registries. The user and token we set for our internal source control won't work for those.